src/client/client.ts

```diff
@@ -0,0 +1,33 @@
+const API_URL = 'https://mate.academy/students-api';
+
+function wait(delay: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
+}
+
+export const getTodos = <T>(url: string): Promise<T> => {
+  return wait(100)
+    .then(() => fetch(API_URL + url, { method: 'GET' }))
+    .then(response => response.json());
+};
+
+export const deleteTodo = <T>(url: string): Promise<T> => {
+  return wait(100)
+    .then(() => fetch(API_URL + url, { method: 'DELETE' }))
+    .then(response => response.json());
+};
```

Comment:
You do almost the same operations in all functions, create a function to avoid repeating